app/javascript/groceries/Groceries.vue

```diff
@@ -11,13 +11,13 @@
 <script setup lang="ts">
 import { Connection } from '@rails/actioncable';
 import Cookies from 'js-cookie';
-import { get } from 'lodash-es';
```

Comment:
`get` is not type-safe (it's return value is `any`), so removing its usage seems like a win.